types/snapsvg/index.d.ts

```diff
@@ -279,6 +279,7 @@ declare namespace Snap {
         image(src:string,x:number,y:number,width:number,height:number):Snap.Element;
         line(x1:number,y1:number,x2:number,y2:number):Snap.Element;
         path(pathString?:string):Snap.Element;
+        path(pathSpec:(string | number)[][]):Snap.Element;
```

Comment:
I think that should be rewritten as 
```
path(pathSpec?: string | (string | number)[][]):Snap.Element; 
```
That way would be no need to have two signatures

Reply:
Also would be good if you drop piece of code in the issue description in the tests file, so compilation will check you example when new submissions would be added